app/components/ui/LandingPageResourceBlock.jsx

```diff
@@ -4,7 +4,8 @@ import PropTypes from 'prop-types';
 const HOST_QUERY = '/search?query=';
 
 class LandingPageResourceBlock extends Component {
-  render() {
+    render() {
+    const url = `/search?refinementList[categories][0]=${encodeURIComponent(this.props.config.TITLE.BLUE_WORD)}`
```

Comment:
I feel like I see this replicated in several different pages, and I feel like it'd be useful to consolidate it into a single helper function that knows the URLs of various search pages.